x-pack/plugins/index_lifecycle_management/public/application/services/navigation.ts

```diff
@@ -4,6 +4,20 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export const getPolicyPath = (policyName: string): string => {
+export const ROUTES = {
+  list: '/policies',
+  edit: '/policies/edit/:policyName?',
+  create: '/policies/edit',
+};
+
+export const getPolicyEditPath = (policyName: string): string => {
   return encodeURI(`/policies/edit/${encodeURIComponent(policyName)}`);
 };
+
+export const getPolicyCreatePath = () => {
+  return `/policies/edit`;
```

Comment:
What do you think of replacing this with `ROUTES.edit`. Otherwise I think we can remove the use of string literals to conform to other string usages.

Reply:
I'd prefer keeping `ROUTES` variable name here for shortness sake and it seemed to me as a good convention to use for a react-router config paths.